minimizer/minimize.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if ! [[ -x "$(command -v dustmite)" ]]; then
+    echo "No DustMite installed. It is typically contained in a package called 'dtools'."
+    exit 1
+fi
+
+available_scripts=()
+for filename in ./conditions/*.sh; do
+    filename=$(basename -- "$filename")
+    filename="${filename%.*}"
+    available_scripts+=("$filename")
+done
+
+if [[ $# -lt 2 ]]; then
+    echo "usage: $0 <file> <condition>"
+    echo "available conditions are:"
+    printf '\t%s\n' "${available_scripts[@]}"
+    exit 1
+fi
+
+FILE="$1"
+
+if ! test -f "$FILE"; then
+    echo "Input file '$FILE' not found."
+    exit 1
+fi
+
+SCRIPT_PATH=$(cd "$(dirname "${BASH_SOURCE[0]}")" || exit; pwd -P)
+CONDITION_SCRIPT="$2"
+
+if ! test -f "$SCRIPT_PATH/conditions/$CONDITION_SCRIPT.sh"; then
+    echo "Condition script '$SCRIPT_PATH/conditions/$CONDITION_SCRIPT.sh' not found."
```

Comment:
You use `$SCRIPT_PATH/conditions/$CONDITION_SCRIPT.sh`, would it be easier to do `CONDITION_SCRIPT="$SCRIPT_PATH/conditions/$2.sh"` instead?